### PR TITLE
Add SLO examples

### PR DESCRIPTION
**Pull request by lootag**

### CODE REVIEW

File: rust/examples/slos/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#[cfg(test)]
+mod tests {
+    use cx_sdk::{
+        auth::ApiKey,
+        client::slos::{ErrorSli, ServiceSlo, SliType, SloClient, SloPeriod, SloStatus},
+        CoralogixRegion,
+    };
+
+    #[tokio::test]
+    async fn slos_test() {
```
Comment (celaus) [inline rust/examples/slos/src/lib.rs:10]: Pls add the more complicated example from the tf provider

File: rust/examples/Cargo.toml [hunk 2]
```diff
@@ -11,6 +11,6 @@ members = [
     "data-sets",
     "dataprime-query", "groups",
     "outgoing-webhooks",
-    "scopes",
+    "scopes", "slos",
     "teams", "users",
```
Comment (celaus) [inline rust/examples/Cargo.toml:15]: fmt (use vscode regular formatter)

File: go/examples/slo_test.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	"context"
+	cxsdk "coralogix-management-sdk/go"
+	v1 "coralogix-management-sdk/go/internal/coralogixapis/apm/services/v1"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+)
+
+func TestExamples(t *testing.T) {
```
Comment (celaus) [inline go/examples/slo_test.go:13]: Same here, please use the complicated example from the tf provider as blue print

**Review by celaus — APPROVED** [review]

File: rust/examples/slos/src/lib.rs (hunk 1 above)
Comment (lootag) [reply rust/examples/slos/src/lib.rs:10]: Done

File: rust/examples/Cargo.toml (hunk 2 above)
Comment (lootag) [reply rust/examples/Cargo.toml:15]: Done

File: go/examples/slo_test.go (hunk 3 above)
Comment (lootag) [reply go/examples/slo_test.go:13]: Done :-)

**Pull request merged**